manifests/osfamily/aix.pp

```diff
@@ -41,21 +43,28 @@
       $aix_ver_number = '7.1'
     } else {
       # 6.19.1 is the last puppet6 release that ships AIX 6.1 packages
+
       $aix_ver_number = versioncmp($puppet_agent::prepare::package_version, '6.19.1') ? {
         1       => '7.1',
         default => '6.1'
       }
     }
   }
 
+  if (versioncmp($pe_server_version, '2021.7.7') < 0) {
+    $aix_class_name = "aix-${aix_ver_number}-power"
+  } else {
+    $aix_class_name = 'aix-power'
+  }
```

Comment:
I like to avoid repeating the variable assignment for if/else statements like these (also removed the extra parentheses):

```suggestion
  $aix_class_name = if versioncmp($pe_server_version, '2021.7.7') < 0 {
    "aix-${aix_ver_number}-power"
  } else {
    'aix-power'
  }
```